sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/npm-log-level/build.sbt

```diff
@@ -0,0 +1,21 @@
+name := "npm-log-level"
+enablePlugins(ScalaJSBundlerPlugin)
+scalaVersion := "2.11.12"
+webpackBundlingMode := BundlingMode.LibraryOnly()
+emitSourceMaps := false
+npmDependencies in Compile += "http-aws-es" -> "6.0.0"
+
+val assertWarning = taskKey[Unit]("checks npm log  warn")
+assertWarning := check(true).value
+
+def check(expectation: Boolean) = Def.task[Unit] {
+  val lastLog: File = BuiltinCommands.lastLogFile(state.value).get
+  val last: String = IO.read(lastLog)
+  val contains = last.contains("[warn] npm")
+  if (expectation && !contains) {
+    sys.error("Expected `[warn] npm`, but not found")
+  } else {
+    IO.write(lastLog, "") // clear the backing log for for 'last'.
```

Comment:
Why do we need to overwrite the log file?

Reply:
Actually, this is copy from sbt's test https://github.com/sbt/sbt/blob/cc0dc0c9037eea25ff77d1081a9c4584823fcab3/sbt/src/sbt-test/reporter/nowarn/build.sbt#L9-L20
I think the original sbt's test requires clearing log, since it check content of log twice (sub1 and sub2 project).

However, this is not applicable to this PR.
So I removes the overwriting